pom.xml

```diff
@@ -0,0 +1,621 @@
+      <name>closure-templates-discuss</name>
+      <subscribe>closure-templates-discuss+subscribe@googlegroups.com</subscribe>
+      <unsubscribe>closure-templates-discuss+unsubscribe@googlegroups.com</unsubscribe>
+      <post>closure-templates-discuss@googlegroups.com</post>
+      <archive>https://groups.google.com/forum/#!forum/closure-templates-discuss</archive>
+    </mailingList>
+  </mailingLists>
+
+  <scm>
+    <connection>scm:git:https://github.com/google/closure-templates.git</connection>
+    <developerConnection>scm:git:git@github.com:google/closure-templates.git</developerConnection>
+    <url>https://github.com/google/closure-templates.git</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <java.version>1.7</java.version>
+    <guice.version>3.0</guice.version>
+    <soy.examples>${project.basedir}/examples</soy.examples>
+    <soy.examples.out>target/examples</soy.examples.out>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <version>1</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject</groupId>
+      <artifactId>guice</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject.extensions</groupId>
+      <artifactId>guice-assistedinject</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject.extensions</groupId>
+      <artifactId>guice-multibindings</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>14.0</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.ibm.icu</groupId>
+      <artifactId>icu4j</artifactId>
+      <version>51.1</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>aopalliance</groupId>
+      <artifactId>aopalliance</artifactId>
+      <version>1.0</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>args4j</groupId>
+      <artifactId>args4j</artifactId>
+      <version>2.0.23</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <version>2.0.3</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.ant</groupId>
+      <artifactId>ant</artifactId>
+      <version>1.8.3</version>
+      <scope>provided</scope>
```

Comment:
Ah, hadn't thought about the impact on shaded artifacts, good point.

The reason of my comment was that as a user of Closure Templates, if I go read the POM I could mistakenly think that I *have* to provide the Ant dependency for Soy to work. Having an additional `<optional>true</optional>` would clear that up (from my POV) by saying it's not needed at runtime, “unless you actually need it”. A comment pointing at the `maven-antrun-plugin` would also make it clear that it's a build-time-only dependency and we're using the `provided` scope because Maven has nothing better to propose.